specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -200,6 +215,11 @@ export function Notifications({
               />
             </ErrorBoundary>
           ))}
+          {notifications.length > 0 && (
+            <Button.Fancy onClick={(): void => onClearAll()}>
+              {'Clear All'}
```

Comment:
Make sure to localize this string!
We already have a `Clear all` string localized that you can use. 

https://github.com/specify/specify7/blob/2a259ca22b439ba71b9313705cb2a253ac8edd6e/specifyweb/frontend/js_src/lib/localization/common.ts#L88-L95